junit5/src/main/java/com/linecorp/armeria/internal/testing/TimeoutDebugUtil.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.internal.testing;
+
+import java.lang.management.ManagementFactory;
+import java.util.List;
+
+final class TimeoutDebugUtil {
+
+    private static final boolean disableTimeout = "true".equals(System.getenv("DISABLE_TIMEOUT"));
```

Comment:
Since it is effective only in debug mode.
```suggestion
    private static final boolean disableTimeout = "true".equals(System.getenv("DISABLE_TIMEOUT_ON_DEBUG"));
```

Reply:
Can we use `junit-platform.properties` for JUnit5 instead?

https://github.com/line/armeria/blob/ad0350c15ea098d897657cc79df461ac1aded898/testing-internal/src/main/resources/junit-platform.properties#L2

Reply:
> Can we use junit-platform.properties for JUnit5 instead?

This just prevents junit from invoking thread interrupt when a certain time period passes. The problem in hand is that a server/client timeout occurs while debugging step by step through code in debug mode.